src/roswire/common/package.py

```diff
@@ -116,116 +69,19 @@ class PackageDatabase(ABC, Mapping[str, Package]):
         and `db['foo'] = bar`).
     """
 
-    @classmethod
-    def build(
-        cls, app_instance: "AppInstance", paths: Optional[List[str]] = None
-    ) -> "PackageDatabase":
-        if paths is None:
-            paths = cls._determine_paths(app_instance)
-        db_package = cls._from_paths(app_instance, paths)
-        return db_package
+    _paths_in_package: List[str]
```

Comment:
If this is intended as a protected attribute, it should move to the same location as the other attributes. Also, does the type annotation need to be `List`? Or is `Sequence` or `Collection` sufficient?

Reply:
The property paths() accesses this, so I don't think I can move it. (Or I need to duplicate it down in all subclasses.)